wiki/People/Quality_Assurance_Team/Blog/en.md

```diff
@@ -0,0 +1,71 @@
+
+If you have an idea for a topic to be discussed in the blog, feel free to contact any member of the QAT!
+
+## Current contributors
+
+- ![][flag_US] [Blocko](https://osu.ppy.sh/u/4075092)
+- ![][flag_US] [Cryptic](https://osu.ppy.sh/u/3728904)
+- ![][flag_DE] [Feerum](https://osu.ppy.sh/u/4815717)
+- ![][flag_GB] [JBHyperion](https://osu.ppy.sh/u/4879508)
+- ![][flag_HU] [Kurokami](https://osu.ppy.sh/u/260933)
+- ![][flag_DE] [Mao](https://osu.ppy.sh/u/2204515)
+- ![][flag_HK] [Nardoxyribonucleic](https://osu.ppy.sh/u/876419)
+- ![][flag_DE] [Okoratu](https://osu.ppy.sh/u/1623405)
+- ![][flag_DE] [OnosakiHito](https://osu.ppy.sh/u/290128)
+
+## Post Compendium
+
+### 2016
+
+August
+- [Explore the QAT!](http://osuqat.tumblr.com/post/148493430910/explore-the-qat)
+- [Weekly Report #1](http://osuqat.tumblr.com/post/148998065655/weekly-report-1)
+- [Weekly Report #2](http://osuqat.tumblr.com/post/149339716305/weekly-report-2)
+- [Are huge sets a problem?](http://osuqat.tumblr.com/post/149506493485/are-huge-sets-a-problem)
+
+September
+- [Code of Conduct](http://osuqat.tumblr.com/post/149777211965/code-of-conduct)
+- [Weekly Report #3](http://osuqat.tumblr.com/post/150075576015/weekly-report-3)
+- [Tips on creating “Beginner Friendly” easies](http://osuqat.tumblr.com/post/150197939840/tips-on-creating-beginner-friendly-easies)
+- [osu!catch Spread Problem](http://osuqat.tumblr.com/post/150614301470/osucatch-spread-problem)
+
+October
+- [The QAT Gazette #4 (+ Ask Me Anything)](http://osuqat.tumblr.com/post/151525615445/the-qat-gazette-4)
+- [The way we do spreads is actually really weird](http://osuqat.tumblr.com/post/151834390075/the-way-we-do-spreads-is-actually-really-weird)
+- [How the QAT handle Qualified Beatmaps](http://osuqat.tumblr.com/post/152196956290/how-the-qat-handle-qualified-beatmaps)
+
+November
+- [osu!standard BN Examination](http://osuqat.tumblr.com/post/152896720720/osustandard-bn-examination)
+- [BN Test Review (osu!taiko, catch and mania)](http://osuqat.tumblr.com/post/153726142390/bn-test-review-osutaiko-catch-and-mania)
+
+### 2017
+
+February
+- [Best of 2016](http://osuqat.tumblr.com/post/157575479410/best-of-2016)
+
+April
+- [The QAT Gazette #5](http://osuqat.tumblr.com/post/159522014380/the-qat-gazette-5)
+
+May
+- [BN Test Review (osu!taiko, catch and mania) #2](http://osuqat.tumblr.com/post/160532419785/bn-test-review-2)
+
+June
+- [The QAT Gazette #6 (+ Ask Me Anything)](http://osuqat.tumblr.com/post/161544954160/the-qat-gazette-6)
+
+## Upcoming Topics
+
+- Controversial Mania Mapping
+- Modding and why long mods are not automatically better
+
+## External Links
+
+Original "QAT Blog | Blog Post Compendium" forum post: <https://osu.ppy.sh/community/forums/topics/578437>
```

Comment:
don't use `<url-link>`
use this instead:
`-   Original forum post: [QAT Blog | Blog Post Compendium](https://osu.ppy.sh/community/forums/topics/578437) by [Feerum](https://osu.ppy.sh/users/4815717)`